packages/suite/src/middlewares/firmware/firmwareMiddleware.ts

```diff
@@ -66,7 +67,7 @@ const firmware = (api: MiddlewareAPI<Dispatch, AppState>) => (next: Dispatch) =>
 
             if (
                 // these 3 conditions cover any device reconnected in bootloader mode (possibly accidentally)
-                status === 'reconnect-in-normal' &&
+                (status === 'wait-for-reboot' || status === 'reconnect-in-normal') && // 'wait-for-reboot' (BL >= 1.10.0) is just for easier testing, 'reconnect-in-normal' for real world devices that will install intermediary
```

Comment:
This is totally minor but still: can we put the comment *above* the line its commenting on? It's more readable especially with a long comment like this one.